.gitignore

```diff
@@ -11,8 +11,8 @@ notes/*
 *.beam
 doc/
 
-# leex generated file
-src/string_lexer.erl
+# yecc, leex generated files
+src/*.erl
```

Comment:
great catch, unfortunetly I have spotted them in master too and not rebased this branch

You caught me by surprise. 

Would you mind if I rebased, probably you would have quite some conflicts to resolve :blush: @pareeohnos ?

Reply:
Nah help yourself